packages/repack/android/src/main/java/com/callstack/repack/ScriptConfig.kt

```diff
@@ -33,13 +35,25 @@ data class ScriptConfig(
             val verifyScriptSignature = requireNotNull(value.getString("verifyScriptSignature"))
             val uniqueId = requireNotNull(value.getString("uniqueId"))
 
-            val url = URL(
-                    if (query != null) {
-                        "$urlString?$query"
-                    } else {
-                        urlString
-                    }
-            )
+            val initialUrl = URL(urlString)
+            val initialUri = initialUrl.toURI()
```

Comment:
nit: its almost impossible to distinguish those two variable names 🤣

Reply:
damn, good point 👍